src/Platform/PlatformUnix.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
  */
+#if defined(__unix__) || defined(__APPLE__) || defined(__linux__)
```

Comment:
HI Phobos,
I have heard of sfall yes.  Its a really nice project, however it seems to focus on patching and extending, rather then reimplementing, which I was aiming to do.
Using IDA I was converting fallout back into C code, 1 to 1, simmilar to what KeeperFX did to DungeonKeeper.  Progress was really slow however.
It really seems more usefull to collaborate with you folk on this project!